File: net/Postpony.Api.Sdk/Postpony.Api.Sdk/Model/TrackingRequest.cs
namespace Postpony.Api.Sdk.Model
{
    public class TrackingRequest:ClientRequestBase
    {

        /// <summary>
        /// LabelId
        /// </summary>
        public string LogisticsCode { set; get; }

        public string TrackNo { set; get; }
    }
}

File: java/postpony-api-sdk/src/main/java/com/postpony/model/TrackingRequest.java
package com.postpony.model;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name = "TrackingRequest")
public class TrackingRequest extends  ClientRequestCom {


    public String getLogisticsCode() {
        return logisticsCode;
    }

    public void setLogisticsCode(String logisticsCode) {
        this.logisticsCode = logisticsCode;
    }



    public String getTrackNo() {
        return trackNo;
    }

    public void setTrackNo(String trackNo) {
        this.trackNo = trackNo;
    }

    @XmlElement(name = "TrackNo")
    private String trackNo;

    @XmlElement(name = "LogisticsCode")
    private String logisticsCode;
}
